addons/mail/static/src/chatter/web/form_compiler.js

```diff
@@ -100,6 +100,11 @@ patch(FormCompiler.prototype, {
             return res; // if chatter is inside sheet, keep it there
         }
         const formSheetBgXml = res.querySelector(".o_form_sheet_bg");
+        if (formSheetBgXml) {
+            setAttributes(formSheetBgXml, {
+                "t-attf-class": `{{ __comp__.uiService.size >= ${SIZES.XXL} ? "o_form_chatter_is_aside" : "" }}`,
+            })
+        }
```

Comment:
The JS workaround was initially made to avoid using the `:has` selector, but since we can now use it, there is even simpler: we can detect the `.o-aside` on the next sibiling and it should do the trick. No more JS needed 😄 

Care that initially it was set in the `mail/form_compiler.js` to avoid applying a class if there is no chatter or if the chatter was inside the formsheet (there is a return above my condition) but seeing it now, I should actually have moved this code further down after the o-aside class addition. Anyway, using `:has(+ oe-chatter.o-aside)` without adding any class is better IMO 🙇